.github/workflows/ci.yml

```diff
@@ -26,7 +27,13 @@ jobs:
             os: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
-      - name: Spin up environment
+      - name: Test initialization, with unset PLACEOS_TAG
+        if: ${{ matrix.version == '' }}
+        run: ./placeos start --verbose
+        env:
+          PLACE_EMAIL: robot@place.tech
+          PLACE_PASSWORD: development
+      - name: Test initialization
```

Comment:
This step also needs a conditional, otherwise the init is happening twice.

Reply:
I'll spin it into another job

Reply:
Curse the lack of anchors.. so much duplication. 
The idea here was to try to test both unset and empty